pkg/network/node/ovs/fake_ovs.go

```diff
@@ -297,15 +329,47 @@ func (fake *ovsFake) deleteFlowsHelper(flow string) error {
 	return nil
 }
 
+func (tx *ovsFakeTx) AddGroup(groupID uint32, groupType string, buckets []string) {
+	tx.groups = append(tx.groups, fmt.Sprintf("group add group_id=%d,type=%s,bucket=%s", groupID, groupType, strings.Join(buckets, "bucket=")))
+}
+
+func (tx *ovsFakeTx) DeleteGroup(groupID uint32) {
+	tx.groups = append(tx.groups, fmt.Sprintf("group delete group_id=%d", groupID))
+}
+
 func (tx *ovsFakeTx) Commit() error {
 	var err error
 	if err = tx.fake.ensureExists(); err != nil {
 		return err
 	}
 
+	for _, group := range tx.groups {
+		origGroup := group
+		if strings.HasPrefix(group, "group add") {
+			group = strings.TrimPrefix(group, "group add ")
+			id := strings.TrimPrefix(strings.Split(group, ",")[0], "group_id=")
+			if err != nil {
+				return fmt.Errorf("cannot convert %s to int for group add %s: %v", id, origGroup, err)
```

Comment:
you removed the `strconv.Atoi` but you left the error check if it failed
(likewise below)

Reply:
done